docs/tutorial/intermediate.rst

```diff
@@ -11,105 +12,366 @@ You will learn how to:
 - Configure and run additional pathway reconstruction algorithms on a dataset
 - Enable post-analysis steps to generate post analysis information
 
-Step 1: Transforming Data into SPRAS-Compatible Inputs 
-======================================================
+Step 1: Transforming high throughput experimental data into SPRAS compatible input data
+========================================================================================
+
+1.1 What is the SPRAS-standardized input data?
+-----------------------------------------------
+
+A pathway reconstruction algorithm requires a set of input nodes and an interactome; however, each algorithm expects its inputs to follow a unique format.
 
-1.1 Understanding the Data
--------------------------------------------------------------------
+To simplify this process, SPRAS requires all input data in a dataset to be formatted once into a standardized SPRAS format.
+SPRAS then automatically generates algorithm-specific input files when an algorithm is enabled in the configuration file.
 
-We start with mass spectrometry data containing three biological replicates, each with two technical replicates (IMAC and IP).
-ADD THAT WE COMBINE THESE TOGETHER
-Each replicate measures peptide abundance across multiple time points (0 to 124 minutes).
+.. note::
+    Each algorithm uses the input nodes to guide or constrain the optimization process used to construct reconstruct subnetworks.
 
-Show images and charts as to what is changing instead of giving the code
+    An algorithm maps these input nodes onto the interactome and uses the network to identify connecting paths between the input nodes to form subnetworks.
 
-The goal is to turn this experimental data into the format that SPRAS expects;
-a list of proteins with associated prizes and a defined set of source and target proteins.
 
+Pathway reconstruction algorithms differ in the inputs nodes they require and how they interpret those nodes to identify subnetworks.
 
-1.2 Filtering and Normalizing the Replicates
--------------------------------------------------------------------
+- Some use source and target nodes to defined start and end points. 
```

Comment:
Should we refer to the algorithm docs as a place to describe this? We'll have that after #408.

Reply:
Yes, if we push #408 before the tutorial, I will link to that part of the docs for further information on this.